docs/Home.md

```diff
@@ -1,18 +1,71 @@
-# Welcome to Inductiva API
-
-Inductiva is a Python package for executing large-scale simulations of physical 
-systems directly in the cloud.
-
-Inductiva API offers distinct advantages:
-
-- 🔄 It consolidates various simulation domains, including fluid and molecular 
-dynamics, plasmas, and structural mechanics, under a single unified entry point.
-- 📦 Eliminates the need for installing and managing complex simulation software 
-and corresponding dependencies.
-- 🚀 Allows running hundreds or even thousands of simulations concurrently, with 
-no coding.
-- 💽 Automatically optimizes hardware configurations for each type of simulation 
-(e.g., CPU vs. GPU, appropriate number of CPU cores, RAM, etc.).
-- 🐍 With Inductiva API, you are not limited to a pre-defined GUI or intricate 
-configuration languages and scripts. Instead, you write small Python programs 
-that seamlessly integrate with your existing codebase and ML framework.
+![Inductiva_Logo_noBG(259 x 76 px)](https://github.com/inductiva/inductiva/assets/102880975/c67e31eb-2775-4ee5-ac64-027f4591c0cc)
+# Inductiva: a Python package for running large-scale simulations of physical phenomena on the Cloud.
+
+Inductiva is a Python package designed for researchers, scientists, and engineers 
+focused on leveraging the full potential of [free and open-source physical simulation software packages](). 
+It enables users to perform complex simulations without local hardware constraints, 
+focusing on improving computational efficiency rather than expertise in complex 
+systems designed for scaling simulations. 
+
+This introduction is designed to provide you with an overview on Inductiva API, 
+you will learn:
+
+* [What is the Inductiva API?]()
+* [What are Inductiva API's core functionalities?]()
+* [What can you do?]()
+
+## What is the Inductiva API?
+
+Inductiva API is a Python package that enhances the usability and scalability of 
+[free and open-source physical simulation software packages](), which are pivotal 
+in advancing science and engineering. These software packages, while powerful, often 
+present significant challenges in terms of installation, operation, and particularly, 
+scaling for large-scale computations. This complexity can restrict the potential 
```

Comment:
scaling to large computational infrastructures, such as the Cloud or on-premise clusters.